providers/asrockrack/helpers.go

```diff
@@ -181,7 +181,15 @@ func (a *ASRockRack) createUpdateUser(ctx context.Context, account *UserAccount)
 // at this point all logged in sessions are terminated
 // and no logins are permitted
 func (a *ASRockRack) setFlashMode(ctx context.Context) error {
-	_, statusCode, err := a.queryHTTPS(ctx, "api/maintenance/flash", "PUT", nil, nil, 0)
+	// preserve config is needed by e3c256d4i
+	pConfig := &preserveConfig{PreserveConfig: 1}
```

Comment:
set this configuration based on device model number

Reply:
OK, but this adds common as dependency